RtAudio.h

```diff
@@ -622,11 +622,14 @@ class RTAUDIO_DLL_PUBLIC RtAudio
   typedef pthread_t ThreadHandle;
   typedef pthread_mutex_t StreamMutex;
 
-#else // Setup for "dummy" behavior
+#endif
+
+// Setup for "dummy" behavior if no apis specified.
+#if !(defined(__WINDOWS_DS__) || defined(__WINDOWS_ASIO__) || defined(__WINDOWS_WASAPI__) \
+      || defined(__LINUX_ALSA__) || defined(__LINUX_PULSE__) || defined(__UNIX_JACK__) \
```

Comment:
shouldn't this be `RTAUDIO_API_PULSE`, `RTAUDIO_API_OSS` etc?

Reply:
No, those are only CMake flags, not used in C++.